src/pages/Loans/LoansOverview/components/CollateralModal/CollateralModal.tsx

```diff
@@ -46,10 +46,10 @@ const getContentMap = (t: TFunction, variant: CollateralModalVariant, asset: Loa
     }
   }[variant]);
 
-const getModalVariant = (isCollateralActive: boolean, healthFactor: number): CollateralModalVariant => {
+const getModalVariant = (isCollateralActive: boolean, ltvStatus?: Status): CollateralModalVariant => {
```

Comment:
Just to keep consistency in LTV being capitalized ->
```suggestion
const getModalVariant = (isCollateralActive: boolean, LTVStatus?: Status): CollateralModalVariant => {
```

Reply:
I cannot do that. It should start we lower case.